packages/i18n/src/messages/localizedConstants.tsx

```diff
@@ -1764,3 +1765,139 @@ export const getIndigenousCommunity = (
     indigenousCommunity,
     `Invalid indigenous community '${indigenousCommunity}'`,
   );
+
+const technicalSkillLevels = defineMessages({
+  [SkillLevel.Beginner]: {
+    defaultMessage: "Entry-level",
+    id: "AOJjMk",
+    description: "The technical skill level for beginner",
+  },
+  [SkillLevel.Intermediate]: {
+    defaultMessage: "Intermediate",
+    id: "rt4mVU",
+    description: "The technical skill level for intermediate",
+  },
+  [SkillLevel.Expert]: {
+    defaultMessage: "Advanced",
+    id: "5/C1lJ",
+    description: "The technical skill level for expert",
+  },
```

Comment:
The discrepancy here sticks out to me. Being a bit picky. 
`INTERMEDIATE` maps to "intermediate" and its description mentions intermediate too. Likewise for "Lead". 
This could be troublesome if later on a selection called "Expert" is added in.

Reply:
Are you suggesting we change the enum to be `SkillLevel.Advanced`? I don't think that would be a huge issue. We should ask during stand up tomorrow 👌

Reply:
Yeah, that would look more consistent

Reply:
Yeah, makses sense to me! I think this was just because we created the enum before we had the actual copy 😅

Reply:
As decided in stand up, updated the enum to match the copy. Also, noticed the enum was being sorted for us so I made a helper that returns the skill enum sorted as an array.

[update skill level enum value to match copy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7686/commits/2a6bb929791d46345858cc7f5cca2c966532a82d)